packages/flutter/test/widgets/editable_text_show_on_screen_test.dart

```diff
@@ -489,6 +490,242 @@ void main() {
       // The scroll offset should remain the same.
       expect(controller.offset, 100.0 * 15);
   });
+
+  void testShowCaretOnScreen({ required bool readOnly }) {
+    group('EditableText._showCaretOnScreen, readOnly=$readOnly', () {
+      final TextEditingController textEditingController = TextEditingController();
+      final TextInputFormatter rejectEverythingFormatter = TextInputFormatter.withFunction((TextEditingValue old, TextEditingValue value) => old);
```

Comment:
Does TextInputFormatter need to be imported?  Looking at the [analyzer](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8854478385424369568/+/steps/prepare_environment/0/steps/Analyze/0/steps/analyze/0/stdout).